keras_cv/models/vgg19.py

```diff
@@ -25,39 +25,52 @@
 
 from keras_cv.models import utils
 
+# Copyright 2022 The KerasCV Authors. All Rights Reserved.
```

Comment:
Looks like we accidentally ended up with a second copy of the copyright notice here

Reply:
(and top-level docstring + imports)